assets/js/components/KeyMetrics/ChangeMetricsLink.js

```diff
@@ -69,7 +123,6 @@ export default function ChangeMetricsLink() {
 			>
 				{ __( 'Change metrics', 'google-site-kit' ) }
 			</Link>
-			<SetupCompletedSurveyTrigger />
```

Comment:
I realize this is counter to the IB but I'd prefer if we didn't change this, but also I don't quite see the benefit of pulling so much of this component (which is now unused) into this one. If the in view state is beneficial, we could source that via `useInView` (which should source its state from the `InViewProvider` of the KMW area) although I'm not sure we need to implement an observer just for this button. Pushing the tracking into the render here allows us to use the same conditions for the trigger as for showing. I don't think it's particularly important that the button itself is viewed.